samples/client/petstore/scala-http4s/src/main/scala/org/openapitools/client/apis/PetApi.scala

```diff
@@ -126,7 +126,7 @@ class PetApiEndpointsImpl[F[*]: Concurrent](
     }
   }
 
-  override def getPetById(petId: Long)(implicit auth: _Authorization.ApiKey): F[Pet] = {
+  override def getPetById(petId: Long, auth: _Authorization.ApiKey): F[Pet] = {
```

Comment:
@JennyLeahy thanks for the PR.

Is this a breaking change given that there's a change in the method signature?

Reply:
@JennyLeahy ya, it's breaking change if someone already using the generated client.

Reply:
i wonder if we can add an option for fallback

otherwise, we will need to wait till next major release 8.0.0 which will take a while.